backend/src/main/java/com/htecgroup/skynest/util/JwtUtils.java

```diff
@@ -28,8 +31,13 @@ public class JwtUtils {
   public static final String TOKEN_PREFIX = "Bearer ";
 
   public static long ACCESS_TOKEN_EXPIRATION_MS;
+  public static long EMAIL_TOKEN_EXPIRATION_MS;
   private static Algorithm ALGORITHM;
 
+  private static final String EMAIL_TOKEN_CLAIM = "Email Token";
+  public static final String EMAIL_VERIFICATION_PURPOSE = "verification";
```

Comment:
Does this needs to be in the this util class, or it can be exported in a constants class?

Reply:
I don't think it belongs in the new enum, as I will be using the enum for sending emails, not verification. Does it make sense to make a whole another class for constants instead of leaving it here? @petarstoimenovskihtec

Reply:
If it is used in multiple different places, then constant class is good candidate